### PR TITLE
dumped player map views use same orientation as the map itself

### PR DESCRIPTION
**Pull request by gergely-elias**
ground truth:
ground_truth
player's view without the fix:
Isobel_unflippedmap
player's view with the fix:
Isobel_map

### CODE REVIEW

File: src/supremacy/player.py [hunk 1]
```diff
@@ -191,6 +191,7 @@ def rip(self):
 
     def dump_map(self):
         im = Image.fromarray((self.game_map.array.astype(np.uint8) + 1) * 127)
```
Comment (nvaytet) [inline src/supremacy/player.py:193]: How about just using
```suggestion
        im = Image.fromarray(np.flipud((self.game_map.array.astype(np.uint8) + 1) * 127))
```
?

**Review by nvaytet — COMMENTED** [review]
Thanks for the fix!

**Comment by nvaytet** [comment]
I've also invited you to the new organisation for next year's game ;-)

File: src/supremacy/player.py (hunk 1 above)
Comment (gergely-elias) [reply src/supremacy/player.py:193]: To be honest, I do not know either numpy or pillow good enough to have an educated opinion about possible performance difference of the two approaches. Whichever solution you prefer is fine for me.

**Review by nvaytet — APPROVED** [review]

**Comment by gergely-elias** [comment]
> I've also invited you to the new organisation for next year's game ;-)

Thanks! Looks like I don't have the right to commit though. Or maybe you have another idea for more efficient communication? I've left some messages over discord.

**Pull request merged**